editor/inspector_dock.cpp

```diff
@@ -614,6 +614,26 @@ void InspectorDock::apply_script_properties(Object *p_object) {
 	stored_properties.clear();
 }
 
+void InspectorDock::input(const Ref<InputEvent> &p_event) {
```

Comment:
This should probably use `shortcut_input` (and the corresponding set processing needs to be called in the constructor).